main.py

```diff
@@ -1,10 +1,56 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as source_file:
+        players_data = json.load(source_file)
+
+    for player in players_data:
+        guild_ = None
+        player_guild = players_data[player]["guild"]
+        if player_guild:
+            if not Guild.objects.filter(name=player_guild["name"]).exists():
+                guild_ = Guild(
```

Comment:
In my opinion, using orm instead of class instances and `save` is better

Reply:
In the task there is a sentence 'Create only one instance for each guild, race and skill, do not copy them'. So I thought we needed to use class instances for that